packages/ui-karma-config/lib/index.js

```diff
@@ -21,20 +21,20 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-const path = require('path')
+
 const constants = require('karma').constants
 const choma = require.resolve('choma')
+const filePathCalculator = require('./FilePathCalculator')
 
 const noLaunchers = process.argv.some((arg) => arg === '--no-launch')
 const noHeadless = process.argv.some((arg) => arg === '--no-headless')
 const randomizeTestOrder = process.argv.some((arg) => arg === '--randomize')
 
 const baseWebpackConfig = require('@instructure/ui-webpack-config')
 
-const DEBUG = process.env.DEBUG
+const NO_DEBUG = !!process.env.NO_DEBUG
 const withCoverage = process.env.COVERAGE
```

Comment:
Now tests are made with withe the DEBUG flag by default, so breakpoints can be placed.